src/main/kotlin/de/fraunhofer/iem/spha/cli/commands/ReportCommand.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2024 Fraunhofer IEM. All rights reserved.
+ *
+ * Licensed under the MIT license. See LICENSE file in the project root for details.
+ *
+ * SPDX-License-Identifier: MIT
+ * License-Filename: LICENSE
+ */
+
+package de.fraunhofer.iem.spha.cli.commands
+
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.options.switch
+import de.fraunhofer.iem.spha.cli.SphaToolCommandBase
+import de.fraunhofer.iem.spha.cli.reporter.getMarkdown
+import de.fraunhofer.iem.spha.model.kpi.hierarchy.KpiResultHierarchy
+import java.nio.file.FileSystem
+import java.nio.file.Files
+import kotlin.io.path.createDirectories
+import kotlin.io.path.inputStream
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeFromStream
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+internal class ReportCommand :
+    SphaToolCommandBase(
+        name = "report",
+        help = "Builds a KPI hierarchy from raw values and a hierarchy definition.",
+    ),
+    KoinComponent {
+
+    private val fileSystem by inject<FileSystem>()
+
+    private val resultHierarchy by
+        option(
+                "-r",
```

Comment:
what about an empty parameter for this hierarchy? 
So you call it `spah-cli report YOUR_HIERARCHY --output result.md`